cmd/metricscollector/v1beta1/tfevent-metricscollector/Dockerfile.ppc64le

```diff
@@ -1,5 +1,4 @@
-FROM ibmcom/tensorflow-ppc64le:1.14.0-py3
-RUN pip install rfc3339 grpcio googleapis-common-protos
+FROM ibmcom/tensorflow-ppc64le:2.2.0-py3
```

Comment:
Do you mean that we need to build a container image for ppc64le from [this Dockerfile](https://github.com/kubeflow/katib/blob/702b22d48df3a08ee8745da8534593417681b7ca/cmd/metricscollector/v1beta1/tfevent-metricscollector/Dockerfile)?

Reply:
Yeah, is it possible to re-use the same Dockerfile for `ppc64le` also ?
Or `ibmcom/tensorflow-ppc64le:2.2.0-py3` image has some specific settings for `ppc64le` ?

Reply:
I believe it does not have any specific settings so It is possible to build it from the same Dockerfile. However, we can not use any shell script (ex: `if [ "$(uname -m)" = "ppc64le" ];then echo foo; fi;`).  Therefore if we want to build container images for all CPU arch from the same Dockerfile, we need to use [`ARG`](https://docs.docker.com/engine/reference/builder/#arg) to switch base image.

WDYT? @andreyvelich

Reply:
> I believe it does not have any specific settings so It is possible to build it from the same Dockerfile. 

If it is possible to build from the same Dockerfile, can we extend this condition to support `ppc64le`:
https://github.com/kubeflow/katib/blob/702b22d48df3a08ee8745da8534593417681b7ca/cmd/metricscollector/v1beta1/tfevent-metricscollector/Dockerfile#L13-L18

Reply:
I think it is difficult to switch CPU architectures in that script since TensorFlow for ppc64le is not supported for Python 3.9 so.

Ref: https://powerci.osuosl.org/job/TensorFlow2_PPC64LE_CPU_Release_Build/

Reply:
I see.
In that case, let's keep 2 Dockerfiles.
Does it sound good @gaocegege @johnugeorge ?

Reply:
> In that case, let's keep 2 Dockerfiles.

SGTM.